HackerRank Problem/Ruby/add_array_sum.rb

```diff
@@ -0,0 +1,20 @@
+# Simple add numbers of array
+# Paste the test function in irb and call sum_array_numbers function
+
+def simpleArraySum(numbers)
+  num = numbers.split(" ")
+  num.inject(0) { |sum, i| sum + i.to_i }
+end
+
+fptr = File.open(ENV['OUTPUT_PATH'], 'w')
+
+ar_count = gets.to_i
+
+ar = gets.rstrip
+
+result = simpleArraySum(ar)
+
+fptr.write result
+fptr.write "\n"
+
+fptr.close()
```

Comment:
delete the extra spaced lines after each statements

Reply:
That's how hackerrank has it on their site. Removed it though.

Reply:
Yeah, but that's not a good coding practice. You can see the documentation of any language.